utils/machine/machine.go

```diff
@@ -241,3 +256,42 @@ func addNode(nodes *[]info.Node, id int) (int, error) {
 	}
 	return idx, nil
 }
+
+// s390/s390x changes
+func getMachineArch() (string, error) {
+    	uname := syscall.Utsname{}
+    	err := syscall.Uname(&uname)
+    	if err != nil {
+        	return "", err
+    	}
+
+    	var arch string
+    	for _, val := range uname.Machine {
+        	arch += string(int(val))
```

Comment:
Here we are getting ASCII code from an array. We are converting this ASCII code into string and appending to form the machine hardware name (s390x for zSystem case). Hence we can't use 'Itoa' for this particular case.